README.md

```diff
@@ -69,7 +69,7 @@ Give a ⭐️ if you like this project!
 
```

Comment:
I am suggesting this change here because you have not made changes to the related file

- [ ] Your Calculator component is still class-based whereas you should refactor it to a function component, like so

  ```
  const Calculator = () => {
    ...
  }
  ```
  Ensure everything still works as required after the refactor.

  ![dbdh](https://user-images.githubusercontent.com/36542986/201075552-f075c826-9ac9-4f41-a7ec-85b9b0cfdba3.jpg)